oauth2client/tools.py

```diff
@@ -112,7 +112,7 @@ def log_message(self, format, *args):
 
 
 @util.positional(3)
-def run_flow(flow, storage, flags, http=None):
+def run_flow(flow, storage, flags=argparser.parse_args(), http=None):
```

Comment:
Yes, though I'm not endorsing this change or your use of a default like this.

I'm also not 100% sure but seem to remember that the number of positional args was something that needed to wait for a breaking release.